pymc3/distributions/continuous.py

```diff
@@ -4226,6 +4226,31 @@ class Interpolated(BoundedContinuous):
     ========  ===========================================
     Support   :math:`x \in [x\_points[0], x\_points[-1]]`
     ========  ===========================================
+    
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pymc3 as pm
+    import warnings
+    warnings.simplefilter(action="ignore", category=FutureWarning)
+    from scipy.stats import gamma
+    a = 1.99
+    mean, var, skew, kurt = gamma.stats(a, moments='mvsk')
```

Comment:
I don't think these are needed.

Reply:
```
     from scipy.stats import gamma
     rv = gamma(1.99)
```
does this work instead?

Reply:
yes, sorry I did not realize the comment was confusing. I only commented on the bottom line of the 4 shown, not to all 4, github shows more lines to give some context. if the comment pertains to multiple lines, there should be an extra indication about that.

I only meant that the moments were not used, and therefore it was not necessary to compute nor store them.